src/arch/riscv/interrupts.c

```diff
@@ -28,6 +29,12 @@ void interrupts_arch_init()
         fence_sync();
 
         plic_init();
+
+        /* Initialize ACLINT if supported by the platform*/
+        if(platform.arch.aclint_base != 0) {
+            aclint = (void*) mem_alloc_map_dev(&cpu()->as, SEC_HYP_GLOBAL, INVALID_VA, 
+            platform.arch.aclint_base, NUM_PAGES(sizeof(struct aclint_hw)));
+        }
```

Comment:
To follow the same logic of the plic, maybe this could be encapsulated in a `aclint_init` function.

Reply:
Also, shouldn't we be mapping only the sswi module region?